src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
 body {
+  width: 100%;
+  margin: 0;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.navbar {
+  line-height: 60px;
+}
+
+.list {
   margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+}
+
+.link {
+  color: #000;
+  text-decoration: none;
+  display: block;
+}
+
+.link:hover {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.menu {
+  margin-left: 20px;
+}
+
+.menu:first-child {
+  margin-left: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  background-color: #00acdc;
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  display: block;
+  position: absolute;
+  bottom: 0;
```

Comment:
The `.is-active::after` pseudo-element uses `position: absolute`, which requires a positioned ancestor (usually `position: relative`) to function correctly. Ensure that the parent element of `.is-active` has `position: relative` to avoid potential layout issues.